Best practices/Security guidelines.md

```diff
@@ -32,132 +23,111 @@ services.AddCors(options =>
 }
 ```
 
-To enable CORS access for your application, you can use the following sample:
-
-**Startup.cs**:
+Now that we've defined that, pass these values to the CORS configuration. To do that, we can implement an extension method:
 
 ```c#
-app.UseCors("AppCors");
+public static void SetupCorsRules(
+    this IServiceCollection services, 
+    IConfiguration configuration)
+{
+    var allowedOrigins = configuration
+        .GetSection("AllowedOrigins")
+        .AsEnumerable();
+    string[] origins = allowedOrigins
+        .Select(o => o.Value)
+        .Where(v => !string.IsNullOrEmpty(v))
+        .ToArray();
```

Comment:
Maybe add and empty line between two variables?

Reply:
I've merged these into one since there is no use in having `allowedOrigins` as a separate variable.